perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -551,11 +556,33 @@ def StartBackgroundWorkload(self):
       self.RemoteCommand(
           'nohup sysbench --num-threads=%s --test=cpu --cpu-max-prime=10000000 '
           'run 1> /dev/null 2> /dev/null &' % self.background_cpu_threads)
+    if self.background_network_mbits:
+      self.AllowPort(BACKGROUND_IPERF_PORT)
+      self.RemoteCommand('nohup iperf --server --port %s &> /dev/null &' %
+                         BACKGROUND_IPERF_PORT)
+      (stdout, _) = self.RemoteCommand('pgrep iperf -n')
+      self.server_pid = stdout.strip()
+
+      if self.background_network_iptype == vm_util.IpAddressSubset.EXTERNAL:
+        ip_address = self.ip_address
+      else:
+        ip_address = self.internal_ip
+      base_iperf_cmd = ('iperf --client %s --port %s --time %s -u -b %sM' %
+                        (ip_address, BACKGROUND_IPERF_PORT,
+                         BACKGROUND_IPERF_SECONDS,
+                         self.background_network_mbits))
+      iperf_cmd = 'nohup ' + base_iperf_cmd + ' &> /dev/null &'
+      self.RemoteCommand(iperf_cmd)
+      (stdout2, _) = self.RemoteCommand('pgrep iperf -n')
```

Comment:
Same comment about parentheses. Personally, I find "stdout2" to be an awkward name. It looks like the first stdout from earlier does not need to be preserved beyond here, so why not reuse the variable?